app/controllers/application.js

```diff
@@ -1,6 +1,24 @@
 import Controller from '@ember/controller';
 import { inject as service } from '@ember/service';
+import { action } from '@ember/object'
 
 export default class ApplicationController extends Controller {
   @service router;
+  
+  @action openExplorerURL() {
+    const explorerURL = "/explorer/cities/NYC"
+    let newURL;
+    if (sessionStorage.length) {
+      let queryParams = new URLSearchParams()
+
+      for (let key in sessionStorage) {
```

Comment:
Looping through every key in sessionStorage like this is likely to introduce bugs down the road. `sessionStorage` is an instance of a class so it has more keys than just the ones you're setting. If you log out the `queryParams.toString()` in your action for navigating to the Explorer page, you'll see it includes params for other things we don't want, like `setItem=function`, `length=3`, etc. Apparently `transitionToRoute` is sanitizing those out but we should explicitly pull just the keys we expect to have and need from sessionStorage here.